src/containers/AdminExternalSystems.tsx

```diff
@@ -180,7 +180,7 @@ class AdminExternalSystems extends Component<Props, State> {
                     onClick={this.makeStartEditExternalSystem(system.id)}
                   />
                   <RaisedButton
-                    label="Pull lists"
+                    label="Get Resources"
                     labelPosition="before"
                     icon={<SyncIcon />}
```

Comment:
This SyncIcon looks like a Download icon!

Reply:
How about `Cached`? I'm searching here: https://material-ui.com/components/material-icons/